### ISSUE TITLE
Doesn't remove asserts when using browserify's debug option

### ISSUE THREAD

**Issue by jfirebaugh** [twada/unassertify#1]
With the math.js example from unassertify's README:

```
$ browserify --version
11.2.0
$ browserify math.js -t unassertify  | grep assert -c
0
$ browserify math.js -t unassertify --debug  | grep assert -c
52
```

**Comment by jfirebaugh** [comment]
I [see that this is as designed](https://github.com/twada/unassertify/blob/a325632718cc09f211948459109075e744da0b06/index.js#L28-L30), but I think this is a design mistake. Browserify's `--debug` option means "generate source maps", which are very useful for production builds in which asserts should be removed. Please don't overload it to also mean "don't remove asserts". Folks who don't want to remove asserts can simply not use `-t unassertify`.

**Comment by twada** [comment]
Hi @jfirebaugh, thank you for your suggestion.

>  Folks who don't want to remove asserts can simply not use -t unassertify.

It makes sense a lot. You are right.
I'll release 2.0.0 that removes `assert`s every time (with source-map adjustment when `--debug` is on).

So please wait for it! I'll cut a 2.0.0 release ASAP.
Thanks

**Comment by mourner** [comment]
@twada thanks for responding so quickly, nice to see this being fixed!

**Comment by twada** [comment]
@jfirebaugh @mourner I just released [unassertify 2.0.0](https://github.com/twada/unassertify/blob/master/CHANGELOG.md#200-2015-09-25).
So please check it out!
Thanks again to @jfirebaugh for your suggestion.

**Comment by jfirebaugh** [comment]
Thank you! :bow:
